homepage/styles.css

```diff
@@ -0,0 +1,142 @@
+html {
+    background-color: grey;
+}
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+.header {
+    width: 100%;
+    height: 500px;
+    color: white;
+    background-image: url("background.jpg");
+    background-repeat: no repeat;
+    background-size: cover;
+    border: 1px solid green;  
+} 
+.nav {
+    height: 50px;
+    width: 100%;
```

Comment:
If this is 100% width with no max-width limitation, what will happen if I view it in a huge monitor?